pkg/cli/login/loginoptions.go

```diff
@@ -136,6 +137,10 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
 	// try to TCP connect to the server to make sure it's reachable, and discover
 	// about the need of certificates or insecure TLS
 	if err := dialToServer(*clientConfig); err != nil {
+		if strings.Contains(err.Error(), "certificate is not trusted") {
```

Comment:
Can't we do this workaround only if the err is not type `x509.CertificateInvalidError`;

```go
if _, ok := err.(x509.CertificateInvalidError); !ok {
if strings.Contains(err.Error(), "certificate is not trusted") {
}
}
```

Otherwise, in the future when problem is fixed. We'll still be wrapping the correct error message to the same.

Reply:
The more the fix gets executed only for Mac the better.